### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Screen Reader: Voice over is not reading rating information to the user

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2684]
### Actual Result:
Voice over is not reading rating information to the user

### Expected Result:
Voice over should be reading rating information to the user such as 4 rated application instead of just reading rated by 16 users. 

### GitHubTags:
#Azure Tools for VS Code;#A11ySev2;#A11yTCS;#BM-Azure Tools for VS Code-MAC-Feb2021;#DesktopApp;#WCAG1.3.1;#MAC; #ScreenReader;#VoiceOver;#Element:Button;#AINotInScope

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Start VoiceOver
2. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till "rating" link and check whether VoiceOver is reading about rated information such as 4 rated or not 

### User Impact:
Screen Reader users will know about rated by 16 users but will not about rating information such as 4 rated if Voice over is not reading rating information to the user

### Attachment for Reference:
[Bug_2684_VoiceOver is not reading rating information to the user.zip](https://github.com/microsoft/vscode-azurefunctions/files/6000946/Bug_2684_VoiceOver.is.not.reading.rating.information.to.the.user.zip)

**Comment by ejizba** [comment]
Our extension does not own or control the marketplace. This would reproduce for any VS Code extension and should be filed directly on VS Code instead: https://github.com/microsoft/vscode/issues
